docker-compose/app.env

```diff
@@ -0,0 +1,15 @@
+LOGLEVEL=1
+AUTH_METHOD_OVERRIDE=NONE_OR_EMAIL
+IS_DEV=1
+#DEV_NETWORK_IP=192.168.1.77
+
+#S3_BUCKET=cdn.dev.toadreader.com
+
+APP_URL=toadreader.com  # FIXME this should be the Expo web URL
+RDS_HOSTNAME=mysql
+RDS_PORT=3306
+RDS_USERNAME=ToadReader
+RDS_PASSWORD=ToadReader
+RDS_DB_NAME=ToadReader
+
+WHITELISTED_EMAILS=someone@toadreader.com
```

Comment:
I've generally used a single .env file with clear variable names, e.g. use a prefix to group variables, and mapped the values to the appropriate service's environment variable, reducing any duplication of values. What is the benefit to using a separate file for each service?

Also I .gitignore the .env file, but provide an example, e.g. .env.dist. This is to prevent accidently committing any environment variable values that should not be, but does require work to keep any changes in .env in sync with .env.dist.

Reply:
Its not my preference, but if there will be a clear separation between files that can contain any details related to an environment not running locally, I'm happy to close this for now. I have used local containers to connect to a staging environment to performs tests, which is why I make exposing secrets difficult.